novawallet.xcodeproj/project.pbxproj

```diff
@@ -23067,7 +23067,7 @@
 			outputFileListPaths = (
 			);
 			outputPaths = (
-				"$PROJECT_DIR/NovaPushNotificationServiceExtension/R.generated.swift",
+				$PROJECT_DIR/NovaPushNotificationServiceExtension/R.generated.swift,
```

Comment:
Why the quotes have been removed?

Reply:
Good question. It wasn't manually for sure.